backend/src/main/java/org/uresti/pozarreal/tools/PaymentTools.java

```diff
@@ -0,0 +1,72 @@
+package org.uresti.pozarreal.tools;
+
+import org.uresti.pozarreal.config.FeeConfig;
+import org.uresti.pozarreal.config.PozarrealConfig;
+import org.uresti.pozarreal.dto.House;
+import org.uresti.pozarreal.dto.PaymentByConcept;
+import org.uresti.pozarreal.model.Payment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.uresti.pozarreal.model.PaymentSubConcept.*;
+
+public class PaymentTools {
```

Comment:
I liked the separation of this responsability

Reply:
the problem is that i'm not inject FeeConfig, how can i do it?

Reply:
I didn't notice that .. thanks for bring that up.

In order to inject FeeConfig here you should avoid using static methods ... spring boot performs the injection when the object is created, the method setYearPayments should belong to an object.

Other available option would be pass the FeeConfig as parameter to setYearPayments and inject FeeConfig in the object calling setYearPayments

Reply:
my first option was create a java bean but i think it didn't work